decidim-accountability/app/views/decidim/accountability/admin/timeline_entries/new.html.erb

```diff
@@ -1,3 +1,5 @@
+<% add_decidim_page_title(t(".title")) %>
+<% add_decidim_page_title(translated_attribute(result.title)) %>
```

Comment:
Same here, the title of the accountability project isn't visible, so let's drop it from the document title 
```suggestion
```